etc/kayobe/kolla.yml

```diff
@@ -12,10 +12,12 @@
 
 # URL of Kolla source code repository if type is 'source'.
 #kolla_source_url:
+kolla_source_url: https://git.openstack.org/openstack/kolla
 
 # Version (branch, tag, etc.) of Kolla source code repository if type is
 # 'source'.
 #kolla_source_version:
+kolla_source_version: stable/queens
```

Comment:
Are we upgrading alt-1 as a whole to queens, or is this only for building Magnum from queens and using that with the existing deployment?

Reply:
Only for building Magnum from queens AFAIK

Reply:
I don't think we want to make this change in that case as if someone builds an image for another service, that'll also be queens.

Reply:
+1

Reply:
Do I need to specify kolla_source_version to run the following?
```
	kayobe overcloud container image pull --kolla-tags magnum
	kayobe overcloud service upgrade --kolla-tags magnum 
```

Reply:
I don't think so. By that stage the image will have been built and pushed so it *shouldn't* matter that kolla is on pike.